include/neml2/tensors/Scalar.h

```diff
@@ -46,6 +46,9 @@ class Scalar : public FixedDimTensor<Scalar>
   identity_map(const torch::TensorOptions & options = default_tensor_options);
 };
 
+// I would think we would pull this over ate::abs, but apparently not
+Scalar abs(const Scalar & a);
```

Comment:
What's `ate::abs`?

Reply:
should be `aten`.  I will clarify the comment.